sklearn/svm/_classes.py

```diff
@@ -877,6 +882,16 @@ def __init__(
         break_ties=False,
         random_state=None,
     ):
+        if probability != "deprecated":
+            warnings.warn(
+                "parameter `probability` will be deprecated in version 1.8, "
+                "use `CalibratedClassifierCV(SVC(), ensemble=False)` instead "
+                "of `SVC(probability=True)`",
```

Comment:
```suggestion
                "Parameter `probability` is deprecated in version 1.8, and will"
                "be removed in 1.10. Use "
                "`CalibratedClassifierCV(SVC(), ensemble=False)` instead of "
                "`SVC(probability=True)`",
```